scripts/install_js_deps.sh

```diff
@@ -3,7 +3,9 @@
 # exit when any command fails
 set -e
 
-opam pin -y js_of_ocaml 5.5.2
+# JSOO versions higher than this break QuickJS (labelled break statements)
```

Comment:
Can we link to either (or both) of the stanc/quickjs issues in this comment, just to make said future testing easier